### PR TITLE
Update master contract address

### PR DESCRIPTION
**Pull request by anxolin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,16 @@ The OWL token and related smart contracts.
...
-  * OWLAirdrop: [https://etherscan.io/token/0x85e9fc8273b1ee9d81f4b800071563096f689863](https://etherscan.io/token/0x85e9fc8273b1ee9d81f4b800071563096f689863)
+* **Mainnet**:   
+  * TokenOWLProxy (address): [https://etherscan.io/token/0x1a5f9352af8af974bfc03399e3767df6370d82e4](https://etherscan.io/token/0x1a5f9352af8af974bfc03399e3767df6370d82e4)
+  * TokenOWL (master contract): [https://etherscan.io/token/0x60fbbd1fb0076971e8060631b5dd895f55ad5ab7](https://etherscan.io/address/0x60fbbd1fb0076971e8060631b5dd895f55ad5ab7)
 * **Rinkeby**:
-  * TokenOWL: [https://rinkeby.etherscan.io/token/0xff9874588b2358704832a380ca9892f0a1c57b40](https://rinkeby.etherscan.io/token/0xff9874588b2358704832a380ca9892f0a1c57b40)
+  * TokenOWL (address): [https://rinkeby.etherscan.io/token/0xff9874588b2358704832a380ca9892f0a1c57b40](https://rinkeby.etherscan.io/token/0xff9874588b2358704832a380ca9892f0a1c57b40)
   * TokenOWLProxy: [https://rinkeby.etherscan.io/token/0xa7d1c04faf998f9161fc9f800a99a809b84cfc9d](https://rinkeby.etherscan.io/token/0xa7d1c04faf998f9161fc9f800a99a809b84cfc9d)
```
Comment (Velenir) [inline README.md]: Do you want to add `(master contract)` here?

File: networks.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
     "1": {
       "events": {},
       "links": {},
-      "address": "0x60fBbd1FB0076971E8060631b5dD895F55aD5ab7",
+      "address": "0x60fbbd1fb0076971e8060631b5dd895f55ad5ab7",
```
Comment (Velenir) [inline networks.json]: Why did it suddenly become not checksum?

File: networks.json [hunk 3]
```diff
@@ -79,4 +79,4 @@
       "transactionHash": "0x36ae77e6f99b1932ae6b20126a11c91bd8f3daac7a5368ebdf04cd91c0bef77d"
     }
   }
-}
+}
```
Comment (bh2smith) [inline networks.json:82]:
```suggestion
}

```

File: networks.json (hunk 2 above)
Comment (bh2smith) [reply networks.json:46]: Yea, perhaps we can just revert this change...
```suggestion
      "address": "0x60fBbd1FB0076971E8060631b5dD895F55aD5ab7",
```

**Review by bh2smith — APPROVED** [review]
This is all fine, but given that there is also a bridged OWL token now, maybe we could include it here as well.

**Pull request merged**